src/ProjectTemplates/test/SpaTemplateTest/SpaTemplateTestBase.cs

```diff
@@ -45,8 +45,10 @@ public class SpaTemplateTestBase : BrowserTestBase
         {
             Project = await ProjectFactory.GetOrCreateProject(key, Output);
 
-            var createResult = await Project.RunDotNetNewAsync(template, auth: usesAuth ? "Individual" : null, language: null, useLocalDb);
-            Assert.True(0 == createResult.ExitCode, ErrorMessages.GetFailedProcessMessage("create/restore", Project, createResult));
+            using (var createResult = await Project.RunDotNetNewAsync(template, auth: usesAuth ? "Individual" : null, language: null, useLocalDb))
```

Comment:
What does dispose do here?

Reply:
(What does disposing the result does I mean). Is it just disposing the process?

Reply:
I've looked at what Process.Dispose does and I don't like it done this way. Could we use an using statement instead?
```using var createResult = await ...`

That disposes the process at the end of the scope instead, which I think is safer. Just because I don't remember if we use the results from the process for anything else.

Reply:
I'm fine with that, I actually had it that way on an earlier commit but switched to this way on the idea it would kill processes sooner and therefore maybe better perf. Hypothetically all these processes should be exited anyway though, so the difference is probably marginal and using statements are less likely to cause unexpected behavior.